rpc.repos

```diff
@@ -0,0 +1,21 @@
+repositories:
+    BoostThreadpool:
+        type: git
+        url: https://github.com/richiware/boost_threadpool.git
```

Comment:
Should we fork this into our organization?

Reply:
&#x1F937; Original repo had errors. I've started in my personal space to check if I will able to make it work. Move this repo to eProsima is a decision to be made.

Reply:
I've gone ahead and forked it into eProsima. I think we should not have dependencies on personal repos.

Reply:
Done